### PR TITLE
Transfers

### PR DESCRIPTION
**Pull request by Peter0892**
Removed pending page and replaced it with Transfers page. It is functional but still without validation.

### CODE REVIEW

**Review by JTalarico — DISMISSED** [review]
Please add modals.

File: src/app/app.module.ts [hunk 1]
```diff
@@ -7,7 +7,8 @@ import { HomePage } from '../pages/home/home';
 import { TabsPage } from '../pages/tabs/tabs';
 import { LoginPage } from '../pages/login/login';
 import { SendMoneyPage } from "../pages/send-money/send-money";
-import { PendingPage } from "../pages/pending/pending";
+import {TransfersPage} from "../pages/transfers/transfers";
```
Comment (ghiobi) [inline src/app/app.module.ts:10]: Please add spaces.

**Review by ghiobi — APPROVED** [review]
Please make the following changes.

**Comment by ghiobi** [comment]
Transfers page complete.

**Review by ghiobi — APPROVED** [review]

**Pull request merged**